main.ts

```diff
@@ -6,7 +6,7 @@ const minutesToMilliseconds = (minutes: number) => minutes * 60 * 1000
 const kupoConfig: SnapshotConfig = {
   name: 'kupo',
   checkSnapshotState: checkKupo,
-  snapshotLocation: './test.txt',
+  snapshotLocation: './test/test.txt',
```

Comment:
We should make this point to https://github.com/mlabs-haskell/cardanow/blob/master/docker-compose.yaml#L32